ui/apps/platform/src/Containers/NetworkGraph/types/topology.type.ts

```diff
@@ -72,7 +74,7 @@ export type ExternalGroupData = {
 export type ExternalEntitiesData = {
     type: 'EXTERNAL_ENTITIES';
     id: string;
-    outEdges: Edge[];
+    outEdges: OutEdges;
```

Comment:
The type was actually incorrect before but wasn't caught because we weren't actually accessing the value in our code. It actually isn't an array but an object with the key as a number index and a value of an object with the `properties` key. Let me know if that changes your opinion about the renaming